### PR TITLE
Compiled changelogs for the year we missed [ci skip]

### PR DESCRIPTION
**Pull request by EtraIV**
## About The Pull Request

Updates the changelogs for all our newfriends to look at, at least for the ones that were formatted properly. (@KangTut @AcapitalA @TheChosenEvilOne better start writing proper fucking CLs, see valid tags in `tools/changelog/tags.yml`)

### CODE REVIEW

**Comment by AcapitalA** [comment]
> write changelog
> get told it doesn't matter
> don't write changelog
> get told it matters

just

**Comment by EtraIV** [comment]
> > write changelog
> > get told it doesn't matter
> > don't write changelog
> > get told it matters
> 
> just

shoulda written it anyways lmao, skill issue
you think the server is just gonna stay on some shitty hostwinds 2gb ram VM forever? it was always gonna be upscaling or fizzling out; thank god we're the former

File: html/changelogs/archive/2022-03.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+2022-03-12:
+  Fantii644:
+  - rscadd: Tasers are now back in the game! They can be purchased in Cargo and will be found in armory
+      (three tasers will spawn there)
+  - balance: Tasers no longer hard stamcrits on one shot, gives some minor stam damage and knockdowns
```
Comment (AcapitalA) [inline html/changelogs/archive/2022-03.yml:5]: It used to hardstun. Stamcrit's a different thing.

Comment (EtraIV) [reply html/changelogs/archive/2022-03.yml:5]: i didnt fucking write this you clown, go over to his PR and complain about it lmao

**Pull request merged**